packages/cli/src/cli-start.ts

```diff
@@ -21,6 +21,7 @@ export async function startCli(cwd = process.cwd(), argv = process.argv, options
     })
     .option('-c, --config [file]', 'Config file')
     .option('-w, --watch', 'Watch for file changes')
+    .option('--update-source', 'Update source files with transformed utilities', { default: true })
```

Comment:
I feel this was a dangerous default we picked. I'd be happy to name it as `--write-transformed` and default to `false`.

Reply:
Thanks!
I've updated the flag and made sure that files transformed with directives transformer are also covered by tests.